syntax = "proto3";

message DefaultErrorResponse {
    string error = 1; 
}

